ns-build/README.md

```diff
@@ -0,0 +1,60 @@
+ns-build
+================
+
+Build a namespace from many viash config files.
+
+## Usage
+
+This action will run `viash ns build`.
+
+We recommend using a Linux or MacOS runner if possible.
+
+### Inputs available
```

Comment:
Perhaps add a general comment to look at the documentation for `viash ns build` on the website to get a more in-depth look at the command?

Reply:
Solved!